addons/base_import_module/models/ir_module.py

```diff
@@ -37,7 +37,7 @@ def _import_module(self, module, path, force=False):
         known_mods_names = {m.name: m for m in known_mods}
         installed_mods = [m.name for m in known_mods if m.state == 'installed']
 
-        terp = load_information_from_description_file(module, mod_path=path)
+        terp = _load_manifest(module, mod_path=path)
```

Comment:
I think the use of `_load_manifest()` deserves an explanation directly in the code

Reply:
If I am not mistaken, it is because the files are temporary (the module comes from a zip file).